CourseGrab/LoginViewController.swift

```diff
@@ -2,21 +2,99 @@
 //  LoginViewController.swift
 //  CourseGrab
 //
-//  Created by Daniel Vebman on 1/26/20.
+//  Created by Reade Plunkett on 1/26/20.
 //  Copyright © 2020 Cornell AppDev. All rights reserved.
 //
 
+import GoogleSignIn
+import SnapKit
 import UIKit
 
-class LoginViewController: UIViewController {
+class LoginViewController: UIViewController, GIDSignInDelegate {
```

Comment:
Can we move the stuff that makes this VC conform to `GIDSignInDelegate` into an extension?